src/makeRobot.js

```diff
@@ -38,7 +38,71 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(val) {
+      if (val < 0) {
```

Comment:
```suggestion
      if (val > 0) {
        this.coords.y += val;
      }
```
Всеодно ж в кінці повертаємо this якщо умова не виконалась, а зараз виходить зайва перевірка

Reply:
Так все просто, що аж соромно)

Reply:
Буває)